CMakeLists.txt

```diff
@@ -11,21 +19,20 @@ option(KokkosFFT_INTERNAL_Kokkos "Build internal Kokkos instead of relying on ex
 option(KokkosFFT_ENABLE_BENCHMARK "Build benchmarks for KokkosFFT" OFF)
 
 # Version information
-set(KokkosFFT_VERSION_MAJOR 0)
-set(KokkosFFT_VERSION_MINOR 0)
-set(KokkosFFT_VERSION_PATCH 00)
-set(KokkosFFT_VERSION "${KokkosFFT_VERSION_MAJOR}.${KokkosFFT_VERSION_MINOR}.${KokkosFFT_VERSION_PATCH}")
-
-#Set variables for config file
-math(EXPR KOKKOSFFT_VERSION "${KokkosFFT_VERSION_MAJOR} * 10000 + ${KokkosFFT_VERSION_MINOR} * 100 + ${KokkosFFT_VERSION_PATCH}")
+math(EXPR KOKKOSFFT_VERSION "${PROJECT_VERSION_MAJOR} * 10000 + ${PROJECT_VERSION_MINOR} * 100 + ${PROJECT_VERSION_PATCH} * 10 + ${PROJECT_VERSION_TWEAK}")
 math(EXPR KOKKOSFFT_VERSION_MAJOR "${KOKKOSFFT_VERSION} / 10000")
 math(EXPR KOKKOSFFT_VERSION_MINOR "${KOKKOSFFT_VERSION} / 100 % 100")
 math(EXPR KOKKOSFFT_VERSION_PATCH "${KOKKOSFFT_VERSION} % 100")
```

Comment:
Please remove this, just alias to `PROJECT_VERSION_*` if it is not already done by CMake.

Reply:
The question is that Kokkos versioning has numbers like "4.2.00" which seems to me "PROJECT_VERSION_MAJOR. PROJECT_VERSION_MINOR.PROJECT_VERSION_PATCH.PROJECT_VERSION_TWEAK". 

At the same time, it is reduced into "4.2.0" for version comparison. If we simply control "PROJECT_VERSION_MAJOR. PROJECT_VERSION_MINOR.PROJECT_VERSION_PATCH", we can just alias version variables to these